lnbits/static/js/components/extension-settings.js

```diff
@@ -1,18 +1,13 @@
 Vue.component('lnbits-extension-settings-form', {
   name: 'lnbits-extension-settings-form',
-  props: ['options', 'name', 'cancel'],
-  computed: {
-    endpoint: function () {
-      return `/${this.name}/api/v1/settings?usr=${this.usr}`
-    }
-  },
+  props: ['options', 'name', 'adminkey', 'endpoint', 'cancel'],
```

Comment:
- `name` is no longer required

```suggestion
  props: ['options', 'adminkey', 'endpoint', 'cancel'],
```